src/diffusers/models/unet_blocks_flax.py

```diff
@@ -55,7 +55,7 @@ def setup(self):
         self.attentions = attentions
 
         if self.add_downsample:
-            self.downsample = FlaxDownsample2D(self.out_channels, dtype=self.dtype)
+            self.downsamplers_0 = FlaxDownsample2D(self.out_channels, dtype=self.dtype)
```

Comment:
This should be a list like in PyTorch

Reply:
I can see that there is always one element in the list in the PyTorch modeling file, is it expected that this will change in the future to insert more modules?

Reply:
Good point!  @patrickvonplaten why did we use list for downsamplers ?

Reply:
The only solution I see is to Wrap it in a `LayerCollection` module but this goes against what we want to achieve [here](https://github.com/huggingface/diffusers/pull/527#issuecomment-1249313185) - also I agree that this solution adds a lot of boilerplate